src/helpers/dimensions.ts

```diff
@@ -440,7 +442,8 @@ export function generateDimensions(
             return getXirrDimensions(mdObject);
         }
         case VisualizationTypes.PUSHPIN: {
-            return getGeoChartDimensions(mdObject.buckets);
+            const { buckets } = getGeoPushpinBucketsInMdObject(cloneDeep(mdObject));
```

Comment:
should we add default value for `buckets` like this?
https://github.com/gooddata/gooddata-react-components/pull/1373/files#diff-401c51ce84aba39a6e5b2ba4c990ca86R92

Reply:
updated